src/sempr-core.pc.in

```diff
@@ -6,6 +6,6 @@ database=@DATABASE@
 Name: sempr-core
 Description: Core library of SEMPR (Semantic Environment Mapping, Processing and Reasoning)
 Version: 0.1.0
-Requires: libodb libodb-boost libodb-sqlite eigen3 soprano libgeos-dev libgeos++-dev libgeographic-dev
```

Comment:
Since you removed geos and geographic from the "Requires" list, could you add the needed linker flags to "Libs"? I think that's how it should be handled: They are part of the public API so the user needs them, and the documentation of pkg-config says:
> **Libs**: The link flags specific to this package **and any required libraries that don't support pkg-config.** The same rule as Cflags applies here.

Reply:
In 71ebd4a86a58f1e7b72c1087a7887e554b553493 I have add both libs to the libs flag of the pkgconfig file.